source/2-our-operating-process.md

```diff
@@ -33,14 +33,14 @@ We usually set up our boards in the following way:
 
 \vspace{14pt}
 
-![waffle screenshot](http://echobind.s3.amazonaws.com/images/playbook/waffle.pdf)\
+![waffle screenshot](source/waffle.png)\
 
 \vspace{2pt}
 
-As illustrated, features are first placed into the “Backlog” column. The client is expected to help write these features and review and suggest any necessary changes to them. Then, each week, we estimate which features can be completed and move them into the “Ready” column. While we do our absolute best to complete the “Ready” features within the week’s iteration, we cannot guarantee that all work will be finished. Incomplete items from an iteration get moved to the next one. Once development or design work begins on “Ready” features, they are moved to “In Progress.” Once completed, “In Progress” features are reviewed, and once they are accepted, items are marked as “Done.”  For more details about this process, see [Our Code Process](#our-code-process).
+As illustrated, features are first placed into the "Backlog" column. The client is expected to help write these features and review and suggest any necessary changes to them. Then, each week, we estimate which features can be completed and move them into the "Ready" column. While we do our absolute best to complete the "Ready" features within the week's iteration, we cannot guarantee that all work will be finished. Incomplete items from an iteration get moved to the next one. Once development or design work begins on "Ready" features, they are moved to "In Progress." Once completed, "In Progress" features are reviewed, and once they are accepted, items are marked as "Done.""  For more details about this process, see [Our Code Process](#our-code-process).
```

Comment:
"The client is expected to help write ..."

I'm curious if in the long term clients would be interested in a service where we do help them think through writing down features. Most of the time its more like "It should work like X" - which doesn't really communicate the story. Might be an area to explore as a business opportunity.